benchttp/stats.go

```diff
@@ -0,0 +1,40 @@
+package benchttp
+
+import (
+	"time"
+
+	"github.com/benchttp/worker/stats"
+)
+
+// StatsDescriptor contains a computed stats group description information
+type StatsDescriptor struct {
+	ID         string    `json:"id"`
+	Tag        string    `json:"tag"`
+	FinishedAt time.Time `json:"finishedAt"`
+}
```

Comment:
I'm not entirely sure adding this is the responsibility of `worker`.

We could do :
- `server` inserts this with status `in_progress`
- `worker` updates the status to `done`
- if `worker` fails, it updates the status to `fail` or whatever

Obviously this is out of scope, maybe add an issue if we deem it useful ?

Reply:
Noted, it may indeed be interesting to think about it.